src/patterns/equality-information/sexual-orientation/index.njk

```diff
@@ -0,0 +1,93 @@
+---
+title: Sexual orientation equality question
+layout: layout-example.njk
+---
+
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/radios/macro.njk" import govukRadios %}
+{% from "govuk/components/input/macro.njk" import govukInput %}
+
+{% set sexualOrientationHtml %}
+  {{ govukInput({
+    id: "other-description",
+    name: "other-description",
+    type: "text",
+    spellcheck: false,
+    classes: "govuk-!-width-one-half",
+    label: {
+      text: "How would you describe your sexual orientation? (optional)"
+    }
+  }) }}
+{% endset -%}
+
+{{ govukRadios({
+  idPrefix: "sexual-orientation",
+  name: "sexual-orientation",
+  fieldset: {
+    legend: {
+      text: "Which of the following options best describes how you think of yourself?",
+      isPageHeading: true,
+      classes: "govuk-fieldset__legend--l"
+    }
+  },
+  items: [
+    {
+      value: "heterosexual",
+      text: "Heterosexual or straight"
+    },
+    {
+      value: "gay-lesbian",
+      text: "Gay or lesbian"
+    },
+    {
+      value: "bisexual",
+      text: "Bisexual"
+    },
+    {
+      value: "other",
+      text: "Other",
+      conditional: {
+        html: sexualOrientationHtml
+      }
+    },
+    {
+      divider: "or"
+    },
+    {
+      value: "skip",
+      text: "Prefer not to say"
+    }
+  ]
+}) }}
+
+{{ govukRadios({
+  idPrefix: "gender-identity",
+  name: "gender-identity",
+  fieldset: {
+    legend: {
+      text: "Is the gender you identify with the same as your sex registered at birth?",
+      classes: "govuk-fieldset__legend--m"
+    }
+  },
+  items: [
+    {
+      value: "yes",
+      text: "Yes"
+    },
+    {
+      value: "no",
+      text: "No",
+      conditional: {
+        html: genderHtml
+      }
+    },
+    {
+      value: "skip",
+      text: "Prefer not to say"
+    }
+  ]
+}) }}
```

Comment:
I don't think this is meant to be here – it duplicates a question that appears as part of asking about sex and gender identity, and doesn't appear in the draft.